setup/monitoring/index.md

```diff
@@ -6,3 +6,262 @@ sidebar:
+and values indicating whether the call was successful or not. So rather
+than having two metrics, one for successful calls and the other for
+unsuccessful calls, there is a single metric, where the underlying
+monitoring system can filter the successful from unsuccessful as you want
+depending on how you wish to abstract and interpret the data. In practice
+the metrics have many tags providing a lot of granularity and ways in
+which you can aggregate and interpret them. The data model is described
+further in [the Monitoring reference section](/reference/monitoring/).
+
+In practice there are relatively few distinct metric names (a few hundred).
+However when considering all the distinct time-series streams from the
+different label values there are several thousand distinct streams. Some
+metrics are tagged with the application or account they were used on
+behalf of, so the number of streams may grow as the scope of your
+deployment grows. Typically you will be aggregating these dimensions
+together while breaking out along others. The granularity can come in
+handy when it comes time to diagnose problems or investigate for deeper
+understanding of runtime behaviors but you can aggregate across dimensions
+(or parts of dimensions) when you dont care about that level of refinement.
+
+Each microservice defines and exports its own metrics. The
+`spinnaker-monitoring` daemon augments these by identifying which
+microservice they came from (how it does so depends on the backend
+metrics system you are using). There are no global "Spinnaker" metrics.
+There are only individual metrics on the individual instance of
+concrete microservices that, taken together, compose a "Spinnaker" deployment.
+
+
+### Types of Metrics
+
+There are two basic types of metrics currently supported,
+*counters* and *gauges*.
+
+  * __Counters__ are monotonically increasing values over the lifetime of
+    the process. The process starts out with them at 0, then increments
+    them as appropriate. Some counters may increase by 1 each time, such
+    as the number of calls. Other counters may increase by an arbitrary
+    (but non-negative) amount, such as number of bytes.
+
+    Counters are scoped to the process they are in. If you have a counter
+    in each of two different microservice replicas (including a restart),
+    those counters will be independent of one another. Each process only
+    knows about itself. The daemon adds a tag to each data point that
+    identifies which instance it came from so that you can drill down
+    into individual instances if you need. However, typically you will
+    use your monitoring system to aggregate counters across all replicas.
+
+    Counters are useful to determine rates. Given two points in time,
+    the counter differences will be the measurement delta and the
+    delta divided by the time difference will be the rate.
+    (divide by another 1000000 to convert nanoseconds to milliseconds).
```

Comment:
Is a rate at item per millisecond useful? I've only seen metrics measured at item per second.